hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -0,0 +1,21 @@
+---
+rule_files:
+  - /tmp/rules.verify.yaml
+
+group_eval_order:
+  - kubevirt.cnao.rules
+
+tests:
+  - interval: 1m
+    input_series:
```

Comment:
might need also a non zero vector for the tests?

Reply:
Do you mean a test checking that the alert doesn't fire on (for example) `"1 1 1 1 0 1"`?
@sradco is this possible on this unit test infra?

Reply:
or does fire on `1 1 1 1 1` if needed

Reply:
it should only fire if it is 0 for more than 5 consecutive polls
Talked to shirly and there is a negative test scenario, will change, thanks!

Reply:
Done